classes/Provider/YamlConfigDriver.php

```diff
@@ -15,4 +14,9 @@ public function load($filename)
         $config = Yaml::parse(file_get_contents($filename));
         return $config ?: [];
     }
+
+    public function supports($filename)
+    {
+        return (bool) preg_match('#\.ya?ml(\.dist)?$#', $filename);
```

Comment:
Will probably leave this as-is. The error will get caught by php `file_get_contents` if isn't an actual file. The filename is derived from the current environment which is a value object ([`Environment`](https://github.com/opencfp/opencfp/blob/master/classes/Environment.php)).